ocs_ci/ocs/ui/views.py

```diff
@@ -739,6 +750,18 @@
         By.CSS_SELECTOR,
     ),
     "confirm_delete_inside_storageclass": ("//button[text()='Delete']", By.XPATH),
+    "volume_binding_mode": (
+        "//button[@id='storage-class-volume-binding-mode']",
+        By.XPATH,
```

Comment:
Can you try `("button[id='storage-class-volume-binding-mode']", By.CSS_SELECTOR)`?

Reply:
Made this change